app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,6 @@
 class PostsController < ApplicationController
   layout 'standard'
+
   def index
     @user = User.find(params[:user_id])
     @posts = Post.where(author_id: params[:user_id]).order(id: :asc)
```

Comment:
- [ ] As mentioned by the previous reviewer, you should solve N+1 problems here. You should use the includes method for example:
```ruby
@posts = @user.posts.includes(:comments)
```